integration-libs/s4om/.release-it.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This file can be deleted since its no longer needed due to releasing using RBSC

Reply:
noted. thanks @RadhepS